### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by flamingspaz**
Add travis file to deploy master branch to S3

### CODE REVIEW

**Comment by mmmavis** [comment]
Hey @jbuck 

do you mind reviewing this PR? i've never done this before so i might not be a good reviewer :grin:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+- '0.10'
```
Comment (jbuck) [inline .travis.yml]: Can you make this node 4?

**Comment by jbuck** [comment]
@flamingspaz lgtm!

**Comment by flamingspaz** [comment]
Done!

**Comment by mmmavis** [comment]
:tada: :tada: :tada: thanks @flamingspaz @jbuck :dancers: :dancers: :dancers:

**Comment by mmmavis** [comment]
@flamingspaz do we need to have `.travis.yml` checked in to `gh-pages` branch as well? or just `master` is fine?

**Comment by flamingspaz** [comment]
Yep, I probably should have done it in gh-pages, if you could add it in that'd be great. jbuck has flipped the switch so when you next merge to master it should get deployed.

**Comment by mmmavis** [comment]
@flamingspaz no worries I just did :+1:

**Pull request merged**